src/mono/wasm/Makefile

```diff
@@ -83,11 +83,11 @@ app-builder:
 	$(DOTNET) build $(TOP)/src/tasks/WasmAppBuilder
 
 build-tasks:
-	$(DOTNET) build $(TOP)/src/tasks/WasmBuildTasks $(MSBUILD_ARGS)
+	$(DOTNET) build $(TOP)/src/tasks/tasks.proj /p:Configuration=$(CONFIG)
```

Comment:
Why is MSBUILD_ARGS removed here but not for build-packages?

Reply:
Good catch! Added it back.